openlxp_authentication/urls.py

```diff
@@ -0,0 +1,8 @@
+from django.conf.urls import url
+from django.urls import include, path
+from .views import saml_metadata_view
+
+urlpatterns = [
+    path('metadata/samldb/', saml_metadata_view),
```

Comment:
Might be unnecessarily exposing the implementation here -- recommend a URL route more along the lines of `saml/metadata`, `saml/sp/metadata`, etc.